packages/codemods/1.x-2.x/__testfixtures__/computed-decorator.input.js

```diff
@@ -1,15 +1,15 @@
 /* eslint-disable */
-import {connect as Cerebral} from 'cerebral/react';
-import {limitedList} from "./limitedList";
-import {anotherComputed} from "./anotherComputed";
+import {connect as Cerebral} from "cerebral/react"
```

Comment:
`single quote rule`? Or do you mean `bracketSpacing`?
I ran prettier once before excluding it but changed the `bracketSpacing` rule after.
I can fix that.